src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -222,24 +216,25 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 #Region "Windows Form Designer generated code"
         Friend WithEvents labelDescription As Label
         Friend WithEvents linkLabelHelp As VSThemedLinkLabel
-        Friend WithEvents listViewExtensions As MyExtensionListView
-        Friend WithEvents colHeaderExtensionName As ColumnHeader
+        Friend WithEvents DataGridViewExtensions As MyExtensionDataGridView
+        Friend WithEvents colHeaderExtensionName As DataGridViewTextBoxColumn
```

Comment:
These shouldn't need fields unless we use elsewhere. I'm guessing you aren't using designer to generate these? Can you drag a datagridview onto a designer in a temporary project, add some columns, mark the columns as "private" in Properties window and use that style code gen for below in InitializeComponent so we don't need the extra fields.